opensearch-observability/build.gradle

```diff
@@ -166,6 +167,9 @@ def _numNodes = findProperty('numNodes') as Integer ?: 1
 def es_tmp_dir = rootProject.file('build/private/es_tmp').absoluteFile
```

Comment:
I think this can be replaced with `opensearch_tmp_dir`

Reply:
should i remove `es_tmp_dir` then? I have included `opensearch_tmp_dir` in the next line

Reply:
> Can you also add this task to github CI, as other plugins did

@joshuali925 asked me to not add it in the CI as the infra team will add it to their CI

Reply:
you can remove es_tmp_dir and update this line `systemProperty 'java.io.tmpdir', es_tmp_dir.absolutePath`

Reply:
Yeah i'll change it